gm4_liquid_minecarts/data/liquid_minecarts/functions/load_liquid.mcfunction

```diff
@@ -0,0 +1,17 @@
+#@s liquid minecart that can accept the current tank's liquid
+#run from compare_load_liquid and init_liquid
+
+#take all the liquid in the tank (or as much as the minecart can hold) (amount of liquid to take stored in gm4_lm_data)
+#get remaining space in minecart
+scoreboard players operation @s gm4_lm_data = @s gm4_lt_max
+scoreboard players operation @s gm4_lm_data -= @s gm4_lt_value
+#lower lm_data if tank has less liquid than the remaining space
+scoreboard players operation @s gm4_lm_data < @e[type=armor_stand,limit=1,sort=nearest,tag=gm4_liquid_tank] gm4_lt_value
+
+#add liquid to minecart
+scoreboard players operation @s gm4_lt_value += @s gm4_lm_data
+#take liquid from tank
+scoreboard players operation @e[type=armor_stand,limit=1,sort=nearest,tag=gm4_liquid_tank] gm4_lt_buffer -= @s gm4_lm_data

```

Comment:
Since this module isn't controlled by the liquid tanks base, it should force a liquid update on the liquid tank it interacts with. The update applies the buffer and updates display information. 
`liquid_tanks:liquid_value_update`

Reply:
Ah, I was trying to work out how to do that and found that setting the buffer caused the tank to update. Does running the function simply cause the update to happen immediately instead of when LT gets around to it?

Reply:
Yes. On its own, the base will run an update if the buffer is not 0. The buffer is where changes in liquid go before being applied and is used for the "smart" interface.